src/nrniv/Makefile.am

```diff
@@ -159,7 +159,7 @@ nrniv_DEPENDENCIES = \
 ## Header files that must be included in the distribution:
 noinst_HEADERS = ndatclas.h nrnclass.h nrnmenu.h nrnoc2iv.h ppshape.h \
 	rot3band.h rotate3d.h secbrows.h shape.h shapeplt.h singlech.h \
-	linmod.h nrnste.h arraypool.h structpool.h \
+	linmod.h nrnste.h arraypool.h structpool.h partrans.h \
```

Comment:
That is a good question. The answer must be yes because on install there is no install/include/partrans.h but it is filled with many other .h files. Those are intended to be potentially of use in mod files and I can't imagine such a use for partrans.h. As for install/include/coreneuron, I'm not sure why that is installed unless mod2c/nmodl needs them. cmake/NeuronFileLists.cmake has
```
# =============================================================================
# Lists of header files to install
# =============================================================================
set(HEADER_FILES_TO_INSTALL
...
```
So that is where partrans.h would go if it was missing from  the autottools ```noinst_HEADERS``` variable.